docs/glueops-platform-administrator/configuration/export-logs.md

```diff
@@ -0,0 +1,292 @@
+          name: glueops-core-containerd-lua-script-primary # needs to match the name of the script defined earlier
+        timeAsTable: true
+    - parser:
+        alias: parsedata
+        keyName: log
+        parser: 'glueops-json' # see name used in ClusterParser definition
+        reserveData: true
+    - kubernetes:
+        annotations: true
+        kubeCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        kubeTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+        kubeURL: 'https://kubernetes.default.svc:443'
+        labels: true
+        mergeLogTrim: true
+    - nest:
+        addPrefix: kubernetes_
+        nestedUnder: kubernetes
+        operation: lift
+    - modify:
+        rules:
+          - remove: stream
+          - remove: kubernetes_pod_id
+    - nest:
+        nestUnder: kubernetes
+        operation: nest
+        removePrefix: kubernetes_
+        wildcard:
+          - kubernetes_*
+  match: kube.*
+---
+```
+
+## Defining Outputs:
+
+```yaml title="Define filters and parsers"
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterOutput
+metadata:
+  labels:
+    logging.glueops.dev: 'glueops-core-primary' # needs to match the label defined in: ClusterFluentBitConfig
+  name: 'fluent-bit-datadog-output-primary' # Must be unique among all objects of same type
+  namespace: glueops-core-fluent-operator
+spec:
+  matchRegex: (?:kube|service)\.(.*)
+  datadog:
+    host: 'http-intake.logs.us3.datadoghq.com' # change this to the respective region
+    compress: gzip
+    apikey: XXXXXXXXXXXXXXXXXXXXXXXX
+    dd_source: 'nonprod.antoniostacos.onglueops.com' # update this to your cluster name
+    dd_tags: 'captain_domain=nonprod.antoniostacos.onglueops.com,env=nonprod' # comma separated key value pairs
+    tls: true
+---
+```
+
+## Bringing it all together:
+
+
+```yaml title="Define FluentbitConfig this will be used by the FluentBit object and tie things together"
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFluentBitConfig
+metadata:
+  labels:
+    logging.glueops.dev: 'glueops-core-primary' # needs to match the label used in the Fluentbit resource
+  name: glueops-core-fluent-bit-config-primary  # Must be unique among all objects of same type
+  namespace: glueops-core-fluent-operator
+spec:
+  filterSelector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary' # Needs to match the label used in ClusterFilter
+  inputSelector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary' # needs to match the label used in ClusterInput
+  outputSelector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary' # needs to match the label used in ClusterOutput
+  parserSelector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary' # needs to match the label used in ClusterParser
+  service:
+    httpServer: true
+    parsersFile: parsers.conf
+    httpPort: 45072 # see FluentBit metrics port configuration
+    # storage:
```

Comment:
would it be useful to mention why this is commented out and when the user might want to uncomment it?

Reply:
updated